file-formats/sajt/type/zif_aff_sajt_v1.intf.abap

```diff
@@ -57,7 +57,7 @@ INTERFACE zif_aff_sajt_v1
       "! <p class="shorttext">Name</p>
       "! Name of the parameter
       "! $required
-      name  TYPE  c LENGTH 8,
+      name  TYPE  c LENGTH 38,
```

Comment:
This could be an incompatible change if there are systems that cannot store this value. Did you think about this?

Reply:
How does a parameter class-based application jobs is defined? Was it extended by 30 characters, because the corresponding classname is added to the parameter name?

Reply:
OK, I understand. Then I think we can do the field extension. In lower releases you might get syntax errors if you import an object which uses functionality of higher releases. Same as for ABAP code using new statements not supported in lower releases.